### DISCUSSION TITLE
Add a profile for php-fpm

### DISCUSSION THREAD

**Discussion by osevan** [netblue30/firejail#4655] [Q&A] [answered]
10 years old bug discovered in php fpm

https://bugs.php.net/bug.php?id=81026

https://linuxiac.com/10-year-old-php-fpm-vulnerability-discovered/

We need tidy solution and firejail profile for 90 % of web stacks. 

Firejailing php fpm,mariadb and webserver and bringing together every daemon over Unix domain sockets in secure way.

Maybe running unix domain sockets proxy in own Firejail sandbox  - only one idea. 
Any solutions are welcome ...

**Comment by SkewedZeppelin** [comment, marked as answer]
https://github.com/divestedcg/Brace/blob/master/brace/usr/lib/systemd/system/php-fpm.service.d/99-brace.conf
https://github.com/divestedcg/Brace/blob/master/brace/usr/lib/systemd/system/httpd.service.d/99-brace.conf

**Reply by osevan** [reply]
> https://github.com/divestedcg/Brace/blob/master/brace/usr/lib/systemd/system/php-fpm.service.d/99-brace.conf https://github.com/divestedcg/Brace/blob/master/brace/usr/lib/systemd/system/httpd.service.d/99-brace.conf

Thank you very much for one solution. 

I will saves Systemd profile for fpm and http - hope can change with nginx and Apache 


This solution is well and only for Systemd user.

Not everyone using Systemd, but  I know Systemd got own sandbox solution inside.


We need here Firejail solution as priority.
